_data/nav.yml

```diff
@@ -383,6 +383,14 @@
     - url: "#Accessing the Permanent Pen properties"
     - url: "#Changing the Permanent Pen properties"
   - url: "powerpaste"
+    pages:
+    - url: "#paste_as_text"
+    - url: "#powerpaste_word_import"
+    - url: "#powerpaste_html_import"
+    - url: "#powerpaste_block_drop"
+    - url: "#powerpaste_allow_local_images"
+    - url: "#powerpaste_keep_unsupported_src"
+    - url: "#powerpaste_invalid_inline_elements"
```

Comment:
This looks like it needs updating to match the new name as well. Also should these be in alphabetical order, as the rest appear to be in order?

Reply:
kind of important to word_import and html_import next to each other I think, but I could rearrange the rest of them...